heartbeat/awsvip

```diff
@@ -203,7 +207,8 @@ esac
 AWSCLI="${OCF_RESKEY_awscli}"
 SECONDARY_PRIVATE_IP="${OCF_RESKEY_secondary_private_ip}"
 INSTANCE_ID="$(curl -s http://169.254.169.254/latest/meta-data/instance-id)"
-NETWORK_ID="$($AWSCLI --profile $OCF_RESKEY_profile --output json ec2 describe-instances --instance-id ${INSTANCE_ID} | grep -m 1 'eni' | awk -F'"' '{print$4}')"
+MAC_ADDRESS="$(curl -s http://169.254.169.254/latest/meta-data/mac)"
```

Comment:
Does this cause issues if there are multiple interfaces?

Reply:
I've tried to attach multiple eni, only one interface will return, the main eni.

The rest of others, will provided by another entry.

Should be fine here.

Reply:
Great. Thanks for checking.

Reply:
Just created a new EC2 for double confirm, here's the operation log for your reference.

```
$ ip link show
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9001 qdisc pfifo_fast state UP mode DEFAULT group default qlen 1000
    link/ether 06:a0:47:49:71:fa brd ff:ff:ff:ff:ff:ff
3: eth1: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN mode DEFAULT group default qlen 1000
    link/ether 06:bd:7e:b3:cf:84 brd ff:ff:ff:ff:ff:ff

$ echo "$(curl -s http://169.254.169.254/latest/meta-data/mac)"
06:a0:47:49:71:fa

$ echo "$(curl -s http://169.254.169.254/latest/meta-data/network/interfaces/macs/06:a0:47:49:71:fa/mac)"
06:a0:47:49:71:fa

$ echo "$(curl -s http://169.254.169.254/latest/meta-data/network/interfaces/macs/06:bd:7e:b3:cf:84/mac)"
06:bd:7e:b3:cf:84
```

Reply:
Great. That's good to have here as documentation.